packages/ckeditor5-list/docs/features/lists-editing.md

```diff
@@ -59,6 +59,8 @@ ClassicEditor
 	.catch( /* ... */ );
 ```
 
+This feature only works for pasted contents or on data load, it {@link updating/update-to-41#changes-to-list-merging does not support entering adjacent lists via the editor UI}.
+
```

Comment:
I think this is a good place to link the issue from GH as well. Someone may miss the update guide and navigate from the feature guide.